relay/nakama/Dockerfile

```
@@ -3,18 +3,20 @@
+COPY sign /sign
 
-COPY go.mod go.sum ./
 RUN go mod download
 
-COPY . .
 RUN go build --trimpath --buildmode=plugin -o ./plugin.so
 
 FROM heroiclabs/nakama:3.22.0 AS nakama
 
-RUN apt-get -y update && apt-get -y install curl
+RUN apt-get -y update && \
```

Hadolint

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)